packages/components/organisms/f-checkout/src/exceptions/exceptions.js

```diff
@@ -35,8 +36,31 @@ class PlaceOrderError extends Error {
     }
 }
 
+class DefaultGetCheckoutError extends Error {
+    constructor (message, loadedComponent, errorCode) {
+        super(message);
+        this.messageKey = 'errorMessages.pageLoad.description';
+        this.eventToEmit = loadedComponent === CHECKOUT_LOADED_TYPE.checkout ? EventNames.CheckoutGetFailure : EventNames.CheckoutBasketGetFailure;
```

Comment:
I'm not sure I understand this. The custom error is for `GetCheckout`, but then it's used for handling basket failures as well?
This is linked to my question above re the need to handle 403s in the get basket call.

Reply:
So, for logged in users loadCheckout() function fires to load the checkout. For logged out users loadBasket() function fires to load the checkout instead. That's why I had to add it in both functions to make it work for both logged in and guest checkouts